### PR TITLE
Added buttons (export, print now, print later, present). Added column…

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**
…s (ID, checkbox column ). Removed redirect to admin panel root. Changed the position of back button in data grid.

### CODE REVIEW

File: src/components/ColumnConfig.jsx [hunk 1]
```diff
@@ -106,6 +106,14 @@ class ColumnConfig extends Component{
             </div>
             <div className="column-group">
               <div className="column-group-container">
+                <label className="label">
```
Comment (pepcus-pratik) [inline src/components/ColumnConfig.jsx]: Column should not be included in 'Column Config' options

File: src/components/DataGrid.jsx [hunk 2]
```diff
@@ -227,6 +251,29 @@ class DataGrid1 extends Component {
       });
     }
   }
+  handleEditCheckBoxClick(rowData , e){
+    if (e.target.checked ===  true) {
```
Comment (pepcus-pratik) [inline src/components/DataGrid.jsx]:
```suggestion
    if (e.target.checked) {
```

File: src/components/DataGrid.jsx [hunk 3]
```diff
@@ -227,6 +251,29 @@ class DataGrid1 extends Component {
...
+        selectedStudents: [
+          ...this.state.selectedStudents,
+          rowData,
+        ],
+        selectedStudentsCheck: [
+          ...this.state.selectedStudentsCheck,
+          rowData.id,
+        ],
+      });
+    } else if(e.target.checked === false){
```
Comment (pepcus-pratik) [inline src/components/DataGrid.jsx]:
```suggestion
    } else if(!e.target.checked){
```

File: src/components/DataGrid.jsx [hunk 4]
```diff
@@ -227,6 +251,29 @@ class DataGrid1 extends Component {
...
+      const selectedStudents = cloneDeep(this.state.selectedStudents);
```
Comment (pepcus-pratik) [inline src/components/DataGrid.jsx]: Unnecessary variable

File: src/components/DataGrid.jsx [hunk 5]
```diff
@@ -227,6 +251,29 @@ class DataGrid1 extends Component {
...
+      const selectedStudentsCheck = cloneDeep(this.state.selectedStudentsCheck);
```
Comment (pepcus-pratik) [inline src/components/DataGrid.jsx]: Unnecessary variable

File: src/components/DataGrid.jsx [hunk 6]
```diff
@@ -227,6 +251,29 @@ class DataGrid1 extends Component {
...
+      const removedStudent = selectedStudents.filter((item) => item.id !== e.target.name);
```
Comment (pepcus-pratik) [inline src/components/DataGrid.jsx]:
```suggestion
      const removedStudent = this.state.selectedStudents.filter((item) => item.id !== e.target.name);
```

File: src/components/DataGrid.jsx [hunk 7]
```diff
@@ -227,6 +251,29 @@ class DataGrid1 extends Component {
...
+      const removedSelectCheckBox = selectedStudentsCheck.filter((item) => item !== e.target.name);
```
Comment (pepcus-pratik) [inline src/components/DataGrid.jsx]:
```suggestion
      const removedSelectCheckBox = this.state.selectedStudentsCheck.filter((item) => item !== e.target.name);
```

File: src/components/DataGrid.jsx [hunk 8]
```diff
@@ -241,6 +288,10 @@ class DataGrid1 extends Component {
       Edit</button>
     </div>
   );
+  CheckButton = ({ rowData }) => (
+    <div className = "btn-block"><input name={rowData.id} type="checkbox" onChange={(e) => { this.handleEditCheckBoxClick(rowData, e) }} className="btn-grid" checked={this.state.selectedStudentsCheck.includes(rowData.id) ? "checked": ""}/>
```
Comment (pepcus-pratik) [inline src/components/DataGrid.jsx]: Do proper code formatting

File: src/components/DataGrid.jsx [hunk 9]
```diff
@@ -296,6 +347,24 @@ class DataGrid1 extends Component {
...
+                  border: '1px solid #fffefd',
+                  marginRight: '10px',
+
+                  '&:hover': {
+                    color: '#000',
+                    backgroundColor: 'rgb(231, 104, 14)',
+                    transition: '0.3s all',
+                  }
+                }}
+              >Back </Link>
```
Comment (pepcus-pratik) [inline src/components/DataGrid.jsx]:
```suggestion
              >Back</Link>
```

File: src/components/DataGrid.jsx [hunk 10]
```diff
@@ -349,6 +414,20 @@ class DataGrid1 extends Component {
            </div>*/}
         </div>
         {this.redirectToStudentCorrection()}
+        <div className="buttons-wrapper">
```
Comment (pepcus-pratik) [inline src/components/DataGrid.jsx]: Create a component for this

File: src/components/ColumnConfig.jsx (hunk 1 above)
Comment (BhagyashreeBhoyar) [reply src/components/ColumnConfig.jsx]: Done.

File: src/components/DataGrid.jsx (hunk 2 above)
Comment (BhagyashreeBhoyar) [reply src/components/DataGrid.jsx]: Done.

File: src/components/DataGrid.jsx (hunk 3 above)
Comment (BhagyashreeBhoyar) [reply src/components/DataGrid.jsx]: Done.

File: src/components/DataGrid.jsx (hunk 4 above)
Comment (BhagyashreeBhoyar) [reply src/components/DataGrid.jsx]: Done.

File: src/components/DataGrid.jsx (hunk 5 above)
Comment (BhagyashreeBhoyar) [reply src/components/DataGrid.jsx]: Done.

File: src/components/DataGrid.jsx (hunk 6 above)
Comment (BhagyashreeBhoyar) [reply src/components/DataGrid.jsx]: Done.

File: src/components/DataGrid.jsx (hunk 7 above)
Comment (BhagyashreeBhoyar) [reply src/components/DataGrid.jsx]: Done.

File: src/components/DataGrid.jsx (hunk 8 above)
Comment (BhagyashreeBhoyar) [reply src/components/DataGrid.jsx]: Done.

File: src/components/DataGrid.jsx (hunk 9 above)
Comment (BhagyashreeBhoyar) [reply src/components/DataGrid.jsx]: Done.

File: src/components/DataGrid.jsx (hunk 10 above)
Comment (BhagyashreeBhoyar) [reply src/components/DataGrid.jsx]: Done.

File: src/components/IdCardWrapper.js [hunk 11]
```diff
@@ -0,0 +1,24 @@
+import React, {Component} from 'react';
+
+class IdCardWrapper extends Component{
```
Comment (pepcus-pratik) [inline src/components/IdCardWrapper.js]: @BhagyashreeBhoyar, rename the component to `ActionButtonsWrapper` in the next PR

**Pull request merged**